packages/dbml-homepage/docs/.vuepress/config.js

```diff
@@ -18,5 +18,14 @@ module.exports = {
         ''
       ],
     }
-  }
+  },
+  ga: `${require('./secret').ga}`,
+  plugins: [
+    [
+      '@vuepress/google-analytics',
+      {
+        'ga': `${require('./secret').ga}`
```

Comment:
should we put Google Analytics ID here instead of separating to a different file?

Reply:
I think Google Analytics ID is kind of a secret key so by convention i think we should not hard-coded it.